Simplenote/Classes/SPSettingsViewController+Extensions.swift

```diff
@@ -87,6 +87,7 @@ extension SPSettingsViewController: PinLockSetupControllerDelegate {
     func pinLockSetupControllerDidComplete(_ controller: PinLockSetupController) {
         SPTracker.trackSettingsPinlockEnabled(true)
         dismissPresentedViewController()
+        SPPinLockManager.shared.shouldUseBiometry = biometrySwitch.isOn
```

Comment:
I think you don't really need to check on `biometrySwitch.isOn`, it'll always be `true`, if you got into this spot?

If so, perhaps, for readability, we could set it to `= true`?